i14262: Edited the 'intro to che, installing chectl, & running che lo… (#790)
codenvy/che-docs: src/main/pages/che-7/overview/assembly_introduction-to-eclipse-che.adoc

Eclipse Che is a Kubernetes-native IDE and developer collaboration platform. It provides: 

* Centralized developer environments running on Kubernetes 
* Multi-container workspaces for each developer that can be replicated with a single click through Eclipse Che 
factories
* Pre-built stacks with the ability to create custom stacks for any language or runtime  
* Enterprise integration via Keycloak for AD/LDAP  
* Browser-based IDEs; integration with Che-Theia or any other web IDE (like Jupyter) 
* Support of most recent tooling protocols: Language Server Protocol, Debug Adapter Protocol
* Plug-in mechanism compatible with Visual Studio Code extensions
* An SDK for creating custom cloud developer platforms

[id="getting-started-with-che"]
== Getting started with Che

Get started with Eclipse Che by:

* Following the link:{site-baseurl}che-7/running-che-locally[quick-start guide] to get Eclipse Che installed locally on your computer

> * Learning more about Eclipse Che: link:https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che/#what-is-eclipse-che[What is Eclipse Che] and link:{site-baseurl}che-7/high-level-che-architecture[Architecture overview]

Reviewer: don't use absolute links; this is a same-page anchor, just use `#...`

// TODO: link:che-features-and-benefits.html[Features and benefits of Eclipse Che]
* Discovering Eclipse Che capabilities

== Joining the community

Interested in joining the community? Join us on the following channels:

> * *Public chat:* Join the public link:https://mattermost.eclipse.org/eclipse/channels/eclipse-che[Eclipse Che Mattermost] channel to chat with the developers

Reviewer: all of this was correct; periods in lists are not used when the item isn't a sentence; see IBM Style Guide pg. 62

Get started with Eclipse Che by:

* Following the link:{site-baseurl}che-7/running-che-locally[quick-start guide] to get Eclipse Che installed locally on your computer

> * Learning more about Eclipse Che: <<#what-is-eclipse-che>> and link:{site-baseurl}che-7/high-level-che-architecture[Architecture overview]

Reviewer: this is not correct syntax; see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links

Author: To confirm, it should have been `<<what-is-eclipse-che>>` (no #), correct?

To achieve those core goals, Eclipse Che provides:

> * *Workspaces:* Container-based developer workspaces providing all the tools and dependencies needed to code, build, test, run, and debug applications

Reviewer: whole list was correct

Eclipse Che provides browser-based IDEs for its workspaces. The default IDE is built on link:https://github.com/theia-ide/theia[Theia], and it has the following capabilities:

* *Monaco-based editor:* A fast and responsive editor, CodeLens
* **Language Server Protocol:** Language tooling
* *Debug Adapter Protocol:* Debugger tooling
* *Command palette:* Do everything from your keyboard
* *Task support:* Tasks from Visual Studio Code are extended and support Che commands
* *Embedded preview:* Preview your application directly from the IDE, including Markdown preview
* *Customizable layout:* Adapt the layout using drag and drop
* *And more:* Outline view, search, Git

==== Different IDEs for different use cases

> In Eclipse Che, the IDE is completely decoupled, so that it is possible to plug a different IDE into the Che workspaces:

Reviewer: was correct

* It can be based on Eclipse Theia (as it is a framework to build a web IDE), such as Sirius:
+
video::B6aCqywKpyY[youtube]

* It can be a completely different solution, such as Jupyter or Eclipse Dirigible:
+
video::VooNzKxRFgw[youtube]

> There are situations where the default IDE does not cover the use cases of your audience, or the stakeholders might want to use a dedicated tool that covers their needs instead of an IDE. In the traditional Eclipse IDE world, that was done with Rich Client Platform (RCP) applications.

Reviewer: ```suggestion
For situations in which the default IDE does not cover the use cases of the users or to use a dedicated tool instead of an IDE.
```
